templates/sympa.conf.j2

```diff
@@ -0,0 +1,627 @@
+# {{ ansible_managed }}
+###\\\\ Site customization ////###
+
+## domain
+## Main robot hostname
+## Example: domain	domain.tld
+domain	{{ sympa_domain }}
```

Comment:
Why don't you auto-generate the whole file? Would be like 10 lines instead of over 600.

Reply:
because I want the comments to stay in there better than a unreadable config file

Reply:
The comments can automatically be taken from

```
$ man sympa.conf
```

Reply:
Can you at least document

```
$ man sympa.conf
```

?